src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -69,7 +69,8 @@ public PersistentDataStore() {
         String passwordHash = (String) entity.getProperty("password_hash");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
         String bio = (String) entity.getProperty("bio");
-        User user = new User(uuid, userName, passwordHash, creationTime, bio);
+        String language = (String) entity.getProperty("language");
```

Comment:
Another thing to think about: what will this do with old data, like users who were saved without a language.

The solution to this can be to require everybody to delete their old data, but it's something to be aware of.